src/biotite/structure/atoms.py

```diff
@@ -660,7 +673,17 @@ def __init__(self, length):
             self._coord = None
         else:
             self._coord = np.full((length, 3), np.nan, dtype=np.float32)
-    
+
+    def __repr__(self):
+        """Represent AtomArray as a string for debugging."""
+        atoms = ''
+        for i in range(0, self.array_length()):
+            if len(atoms) == 0:
+                atoms = self.get_atom(i).__repr__()
+            else:
+                atoms = atoms + ',\n' + self.get_atom(i).__repr__()
+        return f'array([{atoms}])'
```

Comment:
Could you put each atom on a separate line with indentation? Currently, the output is
```
array([Atom(...)
...
Atom(...)])
```

but I think
```
array([
    Atom(...)
    ...
    Atom(...)
])
```
would be better readable.